src/collectors/postgres/postgres.py

```diff
@@ -54,9 +76,9 @@ def get_default_config(self):
         Return default config.
         """
         config = super(PostgresqlCollector, self).get_default_config()
+
         config.update({
             'path': 'postgres',
-            'host': 'localhost',
```

Comment:
I don't see the `host` config option being used in https://github.com/python-diamond/Diamond/pull/646/commits/e6d03ba7e67b91cf0e2787d7481e9b2a40876a06